### DISCUSSION TITLE
Applicability for Frequency Domain Problems

### DISCUSSION THREAD

**Discussion by frankkemper** [BAMresearch/probeye#95] [General]
Hi,

I'm pretty new to the topic, and the ProbEye Toolbox helps me to learn step by step. So, first: thank you for sharing!

Currently, I'm working with the "Simple bridge model with time-space correlation" example. From my point of view, the field of application is quite similar to what I want to do. This is: an investigation of the gust wind load on a tower structure. The wind is also correlated with time and space. However, in contrast to the bridge example, there are some differences:
1. The correlation of wind is dependent on the frequency (less correlation with increasing frequency). The is a coherence model to describe that
2. The wind loading is induced at fixed locations simultaneously
3. The goal of the inverse model is to estimate the spectral composition of the loading

Data is available as accelerations (or displacements) in the time or frequency domain.

I'm wondering whether the principal procedure could be used for such kind of problem. Especially the definition of correlation seems complicated to me.  I'm also not sure, whether the formulation should still be in the time domain or in the frequency domain (and estimate the spectra directly).

I would be happy and thankful for some comments.

Frank

**Comment by joergfunger** [comment]
Dear Frank,

1. You could certainly define the correlation also in the frequency domain, the only thing that is then different is that your model output is has to have a output sensors related to frequencies. The correlation model would then also be a function of the frequencey instead of time or space. 
2. Again, this depends on your model being either in frequency of time domain. For the frequency domain, you would have to compute in you forward model for given amplitudes of your wind with presribed frequencies (which is stored at the experiment) the corresponding amplitudes at your outputs sensors (where you have data). In the time domain, each time step would correspond to a data point, I guess this would result in very large datasets with huge correlation matrices and the output being as a large as the amount time steps. I would probably opt for the frequency variant, in particular since small changes might result in large errors (shifts) in the time domain. So make sure that you first define your forward model by identifying, what is actually the input and what is measured, and then split the input into parameters to be attached to the experiment and the ones which are global (in case you have only a single experiment, this does not matter so much). The idea is that for different experiments we have e.g. a changing environmental temperature or a different load position, but the forward model is still the same. 
3. The spectral composition would be an input to your model that is to be indentified, thus you would have to specify a prior for that. Vector parameters are not yet included, which means if you would like to identify the loading amplitute for different frequencies, you would have to specify different parameters (one for each frequency).

**Reply by frankkemper** [reply]
Dear Jörg,

thank you very much for this detailed explanation and your recommendations!
With respect to 1) there is no problem, as we have time domain signals which can easily be converted to PSDs. Actually I modified the bridge example already accordingly. For 2) the forward model is a transfer function (includes mechanical admittance). Exactly, it is much more beneficial to do it in the frequency domain with respect to numerical effort. 3) That's what I was struggling with. This seems to me maybe not consistent as I'd like one unique model, which covers the complete frequency range.

With further review, I found a toolbox (Stingray) which provides Baysian PSD modeling (so fitting in the frequency domain using prior assumptions). For the time being, I'd check whether that might be a more suitable solution to my problem.

Again, thank you very much for this quick and profound support.

Frank

**Reply by joergfunger** [reply]
Just a short question, even if you work in the frequency domain, what would be your model discrepancy term (or in a deterministic optimization the objective function)? In that setting, you would somehow have to integrate over the frequency domain or (numerically) evaluate the function at specific frequencies and then add that up (using some quadrature scheme). Alternatively, you could pick certaint relevant frequencies. Isn't that exactly a set of sensors (each related to a frequency)?
